circleguard/gui.pyw

```diff
@@ -294,29 +294,29 @@ def switch_theme(dark):
         dark_p.setColor(QPalette.Disabled, QPalette.ButtonText, Qt.darkGray)
         dark_p.setColor(QPalette.Disabled, QPalette.Highlight, Qt.darkGray)
 
-        app.setPalette(dark_p)
-        app.setStyleSheet("QToolTip { color: #ffffff; "
+        APP.setPalette(dark_p)
+        APP.setStyleSheet("QToolTip { color: #ffffff; "
                           "background-color: #2a2a2a; "
                           "border: 1px solid white; }")
     else:
-        app.setPalette(app.style().standardPalette())
+        APP.setPalette(APP.style().standardPalette())
         updated_palette = QPalette()
         # fixes inactive items not being greyed out
         updated_palette.setColor(QPalette.Disabled, QPalette.ButtonText, Qt.darkGray)
         updated_palette.setColor(QPalette.Disabled, QPalette.Highlight, Qt.darkGray)
         updated_palette.setColor(QPalette.Inactive, QPalette.Highlight, QColor(240, 240, 240))
-        app.setPalette(updated_palette)
-        app.setStyleSheet("QToolTip { color: #000000; "
+        APP.setPalette(updated_palette)
+        APP.setStyleSheet("QToolTip { color: #000000; "
                           "background-color: #D5D5D5; "
                           "border: 1px solid white; }")
 
 
 if __name__ == "__main__":
     # create and open window
-    app = QApplication([])
-    app.setStyle("Fusion")
-    window = WindowWrapper()
-    set_event_window(window)
-    window.resize(600, 500)
-    window.show()
-    app.exec_()
+    APP = QApplication([])
```

Comment:
why?? all caps should be reserved for final constants like strings or ints, or classes that we declare and never touch again. But we're modifying the attributes of the class after declaration - it's not constant anymore! The all caps is a misnomer here.